payment-backend-service/src/test/java/backend/service/service/test/bean/TestPaymentResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package backend.service.service.test.bean;
+
+import backend.service.service.test.constant.TestConstants;
+import backend.template.dto.PaymentResponseDto;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder(setterPrefix = "with", toBuilder = true)
+public class TestPaymentResponseDto extends PaymentResponseDto {
```

Comment:
I strictly recommend you to use Data, AllArgsConstructor, NoArgsConstructor annotations with pojo. And don't forget about ToString(callSuper = true), EqualsAndHashCode(callSuper = true) for pojos which have parent